.github/workflows/ci.yaml

```diff
@@ -69,6 +69,7 @@ jobs:
         uses: chevdor/srtool-actions@v0.7.0
         with:
           chain: ${{ matrix.chain }}
+          tag: 1.66.0
```

Comment:
1.66.0 is a version of what? It's safer to use the same version of `srtool` as used in Polkadot [v0.9.30](https://github.com/paritytech/polkadot/releases/tag/v0.9.30)